### ISSUE TITLE
Accessibility: MAS42A: High Contrast: Under the dropdown button, "Download Stable" and "Download Insiders" links are not as per the high contrast black theme.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#17]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Prerequisite:**
**_1. Change system setting to High contrast Black_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast Black" from the "Choose a theme" dropdown. Click on Apply.

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in edge.
2. Naviagte to the "Download for Windows" below the "Code editing Redefined"  below the header of the page.
3. Click the dropdown button beside the "Download for Windows"  observe the download arrow links color in high contrast black theme.[refer Screenshot]

**Actual Result:**
The download link arrows below "Stable" are visible in green color and below "Insiders" are visible in grey color in high contrast black .

**Expected Result:**
The download link arrows below "Stable" and below "Insiders" should be visible in yellow color in high contrast black .

**User Impact:**
The persons with colour blindness and low vision will not be able to understand that the download arrows are links and may not use the application effeciently.

**Screenshots:**
![download links are not visible as per the hcb](https://cloud.githubusercontent.com/assets/24825441/21679100/1d82c5a8-d368-11e6-8d15-71341005ed80.png)

**Comment by waderyan** [comment]
Fixed bc1a08f752742496169a1f36bb2028a525275a28
